_source/providenceConfiguration/mainConfiguration/media_processing_operation_filters.csv

```diff
@@ -0,0 +1,35 @@
+Filter,Description,Example
+DESPECKLE,"Reduce the speckles within an image. No parameters.",".. code-block:: text
+
+  DESPECKLE { }"
+MEDIAN,"Apply a median filter to the image.",".. code-block:: text
+
+  MEDIAN { radius = 2 }
+"
+SHARPEN,"Use a Gaussian operator of the given radius and standard deviation (sigma).",".. code-block:: text
+
+  SHARPEN {
```

Comment:
We should define what radius and sigma are as we do below, and verify this is actually how it works. There is/was some debate about the specifics of how these parameters work. And maybe suggest some decent values for sharpening?

Reply:
Sure, the point is that I am not sure what decent values are :( I have checked ImageMagick web site, and provided some values.